besu/src/main/java/org/hyperledger/besu/cli/options/stable/EthstatsOptions.java

```diff
@@ -27,6 +28,7 @@ public class EthstatsOptions implements CLIOptions<NetstatsUrl> {
 
   private static final String ETHSTATS = "--ethstats";
   private static final String ETHSTATS_CONTACT = "--ethstats-contact";
+  private static final String ETHSTATS_PEM_TRUST = "--ethstats-pem-trust";
```

Comment:
Not very clear this is a filepath

How about `ethstats-pem-file` or `ethstats-trust-pem-file`?

Reply:
Renamed to `--ethstats-cacert`, which uses commonly used pattern `--cacert` in *nix cli tools to specify the same.

Reply:
-file prefix seemed to be the most popular form used in besu.

Reply:
Agree, we are using the -file suffix elsewhere